### DISCUSSION TITLE
Create project with no database

### DISCUSSION THREAD

**Discussion by notramo** [luckyframework/lucky#1721] [Feature requests & Ideas] [answered]
Is it possible to create a project with no ORM?

**Comment by jwoertink** [comment, marked as answer]
Yup! Here's a few examples:

* https://github.com/luckyframework/website
* https://github.com/edwardloveall/lucky_bits
* https://github.com/stephendolan/lucky_diff

You can see a bit of code here https://luckyframework.org/guides/database/intro-to-avram-and-orms#alternate-orms

In the next release, it'll be even easier since Avram won't be tied directly to Lucky anymore.
